### DISCUSSION TITLE
Feature Request: Options to delete old plot 1 by 1

### DISCUSSION THREAD

**Discussion by kanasite** [madMAx43v3r/chia-plotter#578] [Ideas]
I'm thinking if it's a good idea to specify some parameters that allow (deletion of the old plot) when the new plot format get finalize and the need to re-plot.

**Comment by CryptoBlockchainTechnologies** [comment]
This is an excellent suggestion that would help small farmers having to replot from having to manually delete a plot, start a new plot, manually delete the next plot...etc. All while trying to not make a mistake and accidentally delete a portable plot. It also requires someone to manually start the potter after each plot which kinda of defeats the purpose of running an automated plotter.

Not sure if there is a way of verifying a plot in a directory if it is a portable plot and then deleting if not.

**Reply by kanasite** [reply]
From my current understanding, the default filename are based on the k value and also the time the plots begin
"plot-k32-2021-06-11-09-04-", perhaps the parameter will allow a simple date and HH:mm input, and only delete plots filename that is lesser than that? that way can ensure you won't be deleting plots created after X

**Reply by kanasite** [reply]
I wrote a basic bash script to remove old plots by interval, feel free to check out @CryptoBlockchainTechnologies 

https://gist.github.com/kanasite/2a90cbd419d5426a8ee313ffa43431c4
